entries/f/faucetpay.io.json

```
@@ -0,0 +1,14 @@
+{
+  "FaucetPay": {
+    "domain": "faucetpay.io",
+    "tfa": [
+      "email",
+      "totp"
+    ],
+    "documentation": "https://faq.faucetpay.io/knowledge-base/what-is-2fa-and-how-do-i-enable-it-in-my-account/",
+    "recover": "https://faq.faucetpay.io/knowledge-base/i-accidentally-removed-the-2fa-application-from-my-mobile-what-should-do-i-now/",
```

```suggestion
    "recovery": "https://faq.faucetpay.io/knowledge-base/i-accidentally-removed-the-2fa-application-from-my-mobile-what-should-do-i-now/",
```
The tests are failing because you misspelt recovery 😉

---

🤦